sub.go

```diff
@@ -64,6 +64,10 @@ func (p *SubParams) AppendTo(body *form.Values, keyParts []string) {
 	if p.TrialEndNow {
 		body.Add(form.FormatKey(append(keyParts, "trial_end")), "now")
 	}
+
+	if p.CancelAtPeriodEnd {
+		body.Add(form.FormatKey(append(keyParts, "cancel_at_period_end")), "true")
+	}
```

Comment:
are you sure this is required? The whole form encoding logic should do this automatically and there should not be a need for this.

Reply:
Yep, the tests fail without it, so it appears to not get generated by default.